docs/source/usage/plugins/particleMergerProbalistic.rst

```diff
@@ -0,0 +1,65 @@
+.. _usage-plugins-particleMergerProbalistic:
+
+Particle Merger Probalistic version
+-----------------------------------
+
+Merges macro particles that are close in phase space to reduce computational load.
+Voronoi-based probalistic variative algorithm. The difference between Base Voronoi algorothm
+and probalistic Version in parameters: instead of threshold of spread in position and momentum
+use ratio of deleted particles. 
+
+
+.param file
+^^^^^^^^^^^
+
+In :ref:`particleMerging.param <usage-params-plugins>` is currently one compile-time parameter:
+
+===================== ====================================================================================
+Compile-Time Option   Description
+===================== ====================================================================================
+``MAX_VORONOI_CELLS`` Maximum number of active Voronoi cells per supercell.
+                      If the number of active Voronoi cells reaches this limit merging events are dropped.
+===================== ====================================================================================
+
+.cfg file
+^^^^^^^^^
+
+============================================ ================================================================================================================
+PIConGPU command line option                 Description
```

Comment:
I think this sections is missing some command line parameters.  e.g. momSpreadThreshold, ...